functions-and-scope.js

```diff
@@ -72,8 +117,15 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+let highestValue = -1;
```

Comment:
Het is beter om als default waarde altijd het eerste cijfer van de array te pakken. Cijfers zullen in dit voorbeeld altijd hoger dan -1 zijn, maar voer je functie eens uit met een array als `[-3, -2, -4, -5]`. Dan zal er `-1` uitkomen terwijl die waarde niet eens in de array voorkomt 😉